derror/multierror_test.go

```diff
@@ -0,0 +1,78 @@
+package derror_test
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/datawire/dlib/derror"
+)
+
+func TestMultiError(t *testing.T) {
+	type testcase struct {
+		Input          derror.MultiError
+		ExpectedString string
+		ExpectedIs     []error
+		ExpectedIsNot  []error
+	}
+	testcases := map[string]testcase{
+		"nil": {
+			Input:          derror.MultiError{},
+			ExpectedString: "(0 errors; BUG: this should not be reported as an error)",
+		},
+		"zero": {
+			Input:          derror.MultiError{},
+			ExpectedString: "(0 errors; BUG: this should not be reported as an error)",
```

Comment:
Is there actually value in having both of these? or are you just really implying that there's not really such a thing as a `nil` `MultiError`?

Reply:
Doh! That first one should have had `Input: nil,`.